cmd/oci-image-tool/man/oci-image-tool-create-runtime-bundle.1.md

```diff
@@ -2,7 +2,13 @@
 % OCI Community
 % JULY 2016
 # NAME
-oci-image-tool-create-runtime-bundle \- Create an OCI image runtime bundle
+oci-image-tool-create-runtime-bundle \- Validates an application/vnd.oci.image.manifest.v1+json and unpacks
+its layered filesystem to `DEST/rootfs`, although the target directory
+is configurable with `--rootfs`. See oci-image-tool-unpack(1) for
```

Comment:
Other man-page cross references are marked up with `**{page}**({section})`, although we haven't always been consistent.  See also opencontainers/ocitools#202.